spec/std/array_spec.cr

```diff
@@ -873,6 +873,83 @@ describe "Array" do
     end
   end
 
+  describe "insert_all" do
+    it "inserts with index 0" do
+      a = [2, 3]
+      expected = [0, 1, 2, 3]
+      a.insert_all(0, [0, 1])
```

Comment:
**suggestion:** We should also test the return value of `#insert_all` to be `self`:
```suggestion
      a.insert_all(0, [0, 1]).should be a
```
Ditto all below.